src/processor/map/map_gds.cpp

```diff
@@ -80,6 +80,7 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapGDSCall(LogicalOperator* logica
         logicalOpToPhysicalOpMap.erase(logicalOperator);
     }
 
+    logicalOpToPhysicalOpMap.insert({logicalOperator, gdsCall.get()});
```

Comment:
Let's open an issue on refactoring the mapping to be consistent that we always keep child sink op, instead of parent scan op of the logical operator.